lib/api/bucketPutVersioning.js

```diff
@@ -17,6 +18,36 @@ import services from '../services';
     x-amz-mfa: [SerialNumber] [TokenCode]
  */
 
+function _parseXML(request, log, cb) {
+    if (request.post === '') {
```

Comment:
This logic replicates pre-existing code from routePUT.js. See
https://github.com/scality/S3/pull/631/files#diff-25f86c99d9d8badaa3509e13e8cc8edbL70